tests/test_iqr_team_score.py

```diff
@@ -0,0 +1,31 @@
+"""A file for testing the scoring of a team based on data retrieved from GitHub
```

Comment:
Could the text be underneath and above the quotation marks so that it doesn't touch it on the same line?